Automattic/wp-calypso
# Update marketplace search by author

#### Proposed Changes

* Implement functionality to filter by Author in Marketplace search

#### Testing Instructions

- Enable the feature flag `marketplace-jetpack-plugin-search` for example adding this query parameter to the URL `flags=marketplace-jetpack-plugin-search`
- Go to `/plugins/<your site>` and click on any Plugin
- On the Plugin Details page, click on the author link under the `Developer` label
- Check the plugins listed are filtered by the search term `developer:<clicked developer>`
- Check that the results are indeed filtered by that Author

![CleanShot 2022-06-22 at 11 32 41@2x](https://user-images.githubusercontent.com/3519124/174998315-ed4b631c-ad51-4550-991c-1ee19b6b39d3.png)


Fixes #https://github.com/Automattic/wp-calypso/issues/64801
